examples/rmg/commented/input.py

```diff
@@ -150,6 +150,16 @@
 )
 
 options(
+    	#provides a name for the seed mechanism produced at the end of an rmg run default is 'Seed'
+    name='RMGButane',	
+	#if True every iteration it saves the current model as a libraries/seeds
+	#(and deletes the old one)
+	#Unlike HTML this is inexpensive time-wise
+	#note this one will be generated at the end of a completed run and some incomplete runs even
```

Comment:
Could you rephrase the comment?
"This one will be generated ... even if False" isn't clear.
If "this one" refers to `generateSeeds`, then let's write it explicitly.
If I did understand correctly, then why is the seed generated even if `generateSeeds=False`?